### DISCUSSION TITLE
Is it possible to "reconstruct" file from mounted exports?

### DISCUSSION THREAD

**Discussion by rokj** [minio/minio#11598] [Q&A]
If just hypothetically for some reason minio server brakes, is it possible to reconstruct a bucket or a file from mounted exports?

**Comment by harshavardhana** [comment]
Even if it's hypothetical do not understand the question.

**Reply by rokj** [reply]
Sorry, I was not clear enough. How would I get file from file/s on mounted disks, which where written by minio with EC, without minio?

**Reply by harshavardhana** [reply]
You cannot @rokj  - requires specialized tuning.

**Reply by klauspost** [reply]
@rokj The best I can tell you is to read the source code. While it is possible, it is by no means trivial to cover all use cases.

You can start here and follow the code: https://github.com/minio/minio/blob/11b2220696db691cdb0a5ca4962f8d0bc9320075/cmd/erasure-object.go#L357
